src/bootstrap/lib.rs

```diff
@@ -115,7 +115,10 @@
 
 #![deny(warnings)]
 #![allow(stable_features)]
-#![feature(associated_consts)]
+#![feature(
+    associated_consts,
+    slice_concat_ext,
+)]
```

Comment:
Why expand this into 4 lines 🤔

Reply:
So future changes to the feature list are simpler

Reply:
We usually just write multiple `#![feature]` attributes though